packages/stems/src/components/TokenValueInput/TokenValueInput.module.css

```diff
@@ -10,23 +10,24 @@
   line-height: 17px;
   letter-spacing: 1px;
   text-transform: uppercase;
-  color: #BEC5E0;
+  color: var(--neutral-light-4);
   margin-bottom: 11px;
 }
 
 .input {
   padding: 14px 16px;
   background: rgba(0, 0, 0, 0.05);
-  border: 1px solid #3F415B;
+  border: 1px solid #3f415b;
```

Comment:
nit but is this intentional?

Reply:
interesting, auto save did this... will run stylelint again